app/api/models/meetup_model.py

```diff
@@ -111,4 +111,34 @@ def update_tags(self, meetup_id, meetup_tags):
         self.conn.commit()
         return self.cur.fetchone()
 
+    def collides(self, data):
+        """ Check if meetup collides with another meetup """
```

Comment:
Hello @tirgei , Great job on the changes. But this name `collides`, for a function, brings a lot of confusion. Please rename it to something that suggests that there is a duplicate.

Reply:
Noted